russian/buildings-dep/buildings-dep.md

```diff
@@ -0,0 +1,2 @@
+Вы дозвонились в ЖЭК, хотя и боялись, что вас не поймут. Объяснили ситуацию и попросили помочь. Вас внимательно выслушали, нос казали что все сейчас заняты,
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.